STUDENT_COURSE _MANAGEMENT/src/main/java/com/springboot/demo/studentcourse/enity/Course.java

```diff
@@ -0,0 +1,57 @@
+package com.springboot.demo.studentcourse.enity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.List;
+
+@Entity
+@Table(name = "course")
+@Data
+public class Course {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "title")
+    @NotEmpty(message = "* is required")
```

Comment:
instead of * can you mention the field name because we are not giving * in the html.

Reply:
Done